Sprint-1/exercises/decimal.js

```diff
@@ -7,3 +7,7 @@ const num = 56.5678;
 // Create a variable called roundedNum and assign to it an expression that evaluates to 57 ( num rounded to the nearest whole number )
 
 // Log your variables to the console to check your answers
+
+const wholeNumberPart = Math.floor(56.7891)
+const decimalPart = (34.5678 - Math.floor(34.5678)).toFixed(4)
```

Comment:
It would be nicer if your code can be reused (without any change) to show the "whole number part", "decimal part", and "rounded value" of any number (not just `56.5678`).

Currently, if we change the initial value of `num` to `46.5678`, your code would not be able to produce the correct result.

Can you change your code (to take advantage of the value stored in variables) so that the code can work for any number assigned to `num`?